server/base/src/main/java/org/apache/accumulo/server/util/ListVolumesUsed.java

```diff
@@ -131,7 +113,11 @@ private static void listTable(Ample.DataLevel level, ServerContext context) thro
   }
 
   public static void listVolumes(ServerContext context) throws Exception {
-    listZookeeper(context);
+    try {
+      listTable(Ample.DataLevel.ROOT, context);
+    } catch (UnsupportedOperationException ex) {
```

Comment:
After the listTable method is updated to not use a scanner, I do not think this exception needs to be caught anymore.

Reply:
Thanks, @keith-turner. I will take a look at the listTable method and continue working on this.